static/vue.js/indicators/objective.js

```diff
@@ -81,9 +81,7 @@ new Vue({
         this.description = item.description;
         this.program_id = item.program_id;
         this.parent_id = item.parent_id;
-        this.parent_obj_list = this.filtered_objectives.filter(
-          el => el.id !== item.id
-        );
+        this.parent_obj_list = this.filtered_objectives;
```

Comment:
@michaelbukachi one small request. This filter was there to ensure that user cannot select the current objective as its parent objective. If the same logic can be implemented in some other way that would work as well :)
![image](https://user-images.githubusercontent.com/13760198/77830686-81e42880-712a-11ea-9fd7-0355dc00196f.png)